scala-cli.sh

```diff
@@ -28,11 +28,20 @@ if [ "$(expr substr $(uname -s) 1 5 2>/dev/null)" == "Linux" ]; then
   fi
   CACHE_BASE="$HOME/.cache/coursier/v1"
 elif [ "$(uname)" == "Darwin" ]; then
-  SCALA_CLI_URL="https://github.com/$GH_ORG/$GH_NAME/releases/download/$TAG/scala-cli-x86_64-apple-darwin.gz"
-  CACHE_BASE="$HOME/Library/Caches/Coursier/v1"
+  arch=$(uname -m)
+  if [[ "$arch" == "x86_64" ]]; then
+    SCALA_CLI_URL="https://github.com/$GH_ORG/$GH_NAME/releases/download/$TAG/scala-cli-x86_64-apple-darwin.gz"
+    CACHE_BASE="$HOME/Library/Caches/Coursier/v1"
```

Comment:
`CACHE_BASE` can be declared before the `if-elif-else` clause, not to repeat it, I think

Reply:
Fixed, thanks for the tip.